packages/main/src/plugin/container-registry.ts

```diff
@@ -1312,7 +1312,7 @@ export class ContainerProviderRegistry {
         AttachStdin: true,
         AttachStdout: true,
         AttachStderr: true,
-        Cmd: ['/bin/sh'],
+        Cmd: ['/bin/sh', '-c', '(bash || clear && sh)'],
```

Comment:
maybe that instead of clearing the screen, we check if command exists ?

```suggestion
        Cmd: ['/bin/sh', '-c', 'if command -v bash >/dev/null 2>&1; then bash; else sh; fi'],
```

Reply:
Just tested it, works fine